demos/mandelbrot/mandel-float.cpp

```diff
@@ -0,0 +1,33 @@
+/***************************************************************************
+ *
+ *  Copyright (C) 2017 Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  Codeplay's ComputeCpp SDK
+ *
+ *  mandel.cpp
+ *
+ *  Description:
+ *    Translation unit for the Mandelbrot SYCL kernel.
+ *
+ **************************************************************************/
+
+#include "mandel.hpp"
+
+template <>
+void MandelbrotCalculator::calc<float>() {
+   this->internal_calc<float>();
+}
```

Comment:
Can you run clang-format on these files? There's a couple of other things that look like they need formatted correctly, too. There's a clang-format file in the repo which has the right formatting options set.

Reply:
Done

Reply:
Hmmm, the newlines are still missing at the end of the file. I thought that had been set in the clang-format file properly. Can you add them in?

Reply:
OK it appears that this is not a thing that `clang-format` will do for us, so for now can you add the single newline at the end of the file in?

Reply:
Added those